scripts/find_ufuncs.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+
+def main():
```

Comment:
This is the script I used to generate the big list of ufuncs. Could ultimately be done as part of a build step.